src/sentry/static/sentry/app/components/gridResizable/styles.tsx

```diff
@@ -0,0 +1,274 @@
+  /* By default, a grid item cannot be smaller than the size of its content.
+     We override this by setting min-width to be 0. */
+  position: relative;
+  min-width: 0;
+  height: ${GRID_HEADER_HEIGHT};
+
+  border-bottom: 1px solid ${p => p.theme.borderDark};
+  background: ${p => p.theme.offWhite};
+`;
+export const GridHeadCellButton = styled.div<GridEditableProps>`
+  position: relative;
+  min-width: 24px; /* Ensure that edit/remove buttons are never hidden */
+  display: ${p => (p.isEditing ? '' : '')};
+
+  margin: ${space(1)};
+  padding: ${space(1)} ${space(1.5)};
+  border-radius: ${p => p.theme.borderRadius};
+
+  color: ${p => p.theme.gray2};
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 1;
+  text-transform: uppercase;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  overflow: hidden;
+
+  background-color: ${p => (p.isEditing ? p.theme.offWhite2 : 'none')};
+  /* cursor: grab; */
+
+  &:hover {
+    color: ${p => p.theme.gray3};
+  }
+`;
+export const GridHeadCellResizer = styled.span<GridEditableProps>`
+  position: absolute;
+  top: 0;
+  right: -2px; /* Overlap half of Resizer into the right neighbor */
+  display: ${p => (p.isEditing ? 'block' : 'none')};
+  width: 4px;
+  height: 100%;
+
+  padding: ${space(1.5)} 1px; /* Padding sets the size of ::after  */
+  z-index: ${Z_INDEX_RESIZER};
+  cursor: col-resize;
+
+  &::after {
+    content: ' ';
+    display: block;
+    width: 2px;
+    height: 100%;
+
+    border-left: 1px solid ${p => p.theme.gray2};
+    border-right: 1px solid ${p => p.theme.gray2};
+  }
+`;
+
+/**
+ *
+ * GridHeadCellButtonHover is the collection of interactive elements to add or
+ * move the columns. They are expected to be draggable.
+ *
+ */
+export const GridHeadCellButtonHover = styled.div<GridEditableProps>`
+  position: absolute;
+  top: 0;
+  left: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+
+  color: ${p => p.theme.gray3};
+
+  &:hover {
+    color: ${p => p.theme.gray4};
+  }
+  &:active {
+    color: ${p => p.theme.gray5};
+  }
+`;
+export const GridHeadCellButtonHoverBackground = styled(GridHeadCellButton)`
+  position: absolute;
+  top: 0;
+  left: 0;
+  display: block;
+  width: 100%;
+  height: 100%;
+
+  background-color: ${p => p.theme.offWhite2};
+  margin: 0;
+`;
+export const GridHeadCellButtonHoverButton = styled.div`
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  width: 20px;
+  height: 20px;
+
+  margin: ${space(0.25)};
+  border: 2px solid ${p => p.theme.gray3};
+  border-radius: ${p => p.theme.borderRadius};
+  background-color: ${p => p.theme.offWhite2};
+  opacity: 1;
+
+  font-size: 14px;
+  cursor: pointer;
+
+  &:hover {
+    border: 2px solid ${p => p.theme.gray4};
+  }
+  &:active {
+    border: 2px solid ${p => p.theme.gray5};
+  }
+`;
+export const GridHeadCellButtonHoverDraggable = styled(InlineSvg)`
+  cursor: grab;
```

Comment:
```suggestion
  cursor: grab;
  user-select: none;
```

when the user drags on these draggable handles, we prefer nothing is selected (e.g. no text selection)